### DISCUSSION TITLE
Does your team still provide the official documentation of the previous version of AutoGluon, now there is only version 0.7.0

### DISCUSSION THREAD

**Discussion by imrooki** [autogluon/autogluon#3073] [Q&A] [answered]
Does your team still provide the official documentation of the previous version of AutoGluon, now there is only version 0.7.0

**Comment by shchur** [comment, marked as answer]
You can, for example, access the documentation for v0.6.2 here https://auto.gluon.ai/v0.6.2/index.html. For other versions you just need to replace `v0.6.2` in the URL.

**Reply by imrooki** [reply]
Thank you very much for your reply, the AG model has given me great help, thank you for your team and contributors.
